### ISSUE TITLE
Component-Specific Observations: Reactour

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#748]
A third party component, Reactour, is used to provide the MathShare tutorial experience.  This component is completely inaccessible to screen reader users (and potentially has accessibility problems for other groups of users as well). 
 
Carrying out a full review of third-party components is not within the scope of this document, but the tutorial cannot be considered to be accessible or usable at this point. 
 
This is not the only instance of a third-party module being integrated into the codebase without undergoing accessibility testing — see also: Inaccessible Sort Mechanism in the Home -> Navigation -> Problems component.

**Comment by benetechMatt** [comment]
We will find a replacement for the tour.  In development

**Comment by jscholes** [comment]
@benetechMatt, @johnhbenetech and @alexrcabral - where do we stand on this?  The tutorial is still present inside the help center, and still completely unusable for screen reader users.  Possibly other groups as well, but I can't use it myself so I don't know.

Preparing an alternative may understandably take some time, but do we have a timeline and action plan for what to do until then?  For example, either removing the inaccessible tutorial, or at least indicating to users that it isn't accessible and will be replaced soon.
